docs/tutorial-epsilla.rst

```diff
@@ -1,4 +1,4 @@
-Text embedding
+Tutorial: Text embedding
```

Comment:
Adding this title keeps it consistent with the other tutorials.

Before:
<img width="400" alt="Screenshot 2023-11-24 at 7 22 10 PM" src="https://github.com/adbar/trafilatura/assets/5278006/88f52d53-8200-4a68-a017-2d8ad6015e36">

After:
<img width="335" alt="Screenshot 2023-11-24 at 7 22 38 PM" src="https://github.com/adbar/trafilatura/assets/5278006/9bb372f5-c3ff-4c79-84a5-5580786909bc">